api/server/httputils/form.go

```diff
@@ -14,10 +14,13 @@ import (
 	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
 )
 
+func ParseBool(s string) bool {
+	return !(s == "" || s == "0" || s == "no" || s == "false" || s == "none")
+}
```

Comment:
I guess we have a slightly more strict set of accepted values, but perhaps we should at some point consider using `strconv.ParseBool`; https://pkg.go.dev/strconv#ParseBool

Reply:
I can change it here if you like/okay either way.